projects/kit/components/slider/slider.component.ts

```diff
@@ -1,84 +1,93 @@
-import {DOCUMENT} from '@angular/common';
 import {
     ChangeDetectionStrategy,
-    ChangeDetectorRef,
     Component,
-    forwardRef,
+    ElementRef,
+    HostBinding,
+    HostListener,
     Inject,
+    Input,
     Optional,
     Self,
 } from '@angular/core';
 import {NgControl} from '@angular/forms';
+import {USER_AGENT} from '@ng-web-apis/common';
 import {
-    isNativeFocused,
-    TUI_FOCUSABLE_ITEM_ACCESSOR,
-    TuiFocusableElementAccessor,
-    TuiNativeFocusableElement,
+    CHROMIUM_EDGE_START_VERSION,
+    isEdgeOlderThan,
+    tuiDefaultProp,
 } from '@taiga-ui/cdk';
-import {AbstractTuiSlider} from '@taiga-ui/kit/abstract';
-import {TUI_FROM_TO_TEXTS} from '@taiga-ui/kit/tokens';
-import {Observable} from 'rxjs';
+import {TuiSizeS} from '@taiga-ui/core';
 
-// @dynamic
 @Component({
-    selector: 'tui-slider',
-    templateUrl: '../../abstract/slider/slider.common.template.html',
-    styleUrls: ['../../abstract/slider/slider.common.style.less', './slider.style.less'],
+    selector: 'input[tuiSlider]',
+    template: ``,
+    styleUrls: ['./slider.style.less'],
     changeDetection: ChangeDetectionStrategy.OnPush,
-    providers: [
-        {
-            provide: TUI_FOCUSABLE_ITEM_ACCESSOR,
-            useExisting: forwardRef(() => TuiSliderComponent),
-        },
-    ],
 })
-export class TuiSliderComponent
-    extends AbstractTuiSlider<number>
-    implements TuiFocusableElementAccessor
-{
-    constructor(
-        @Optional()
-        @Self()
-        @Inject(NgControl)
-        control: NgControl | null,
-        @Inject(ChangeDetectorRef) changeDetectorRef: ChangeDetectorRef,
-        @Inject(DOCUMENT) documentRef: Document,
-        @Inject(TUI_FROM_TO_TEXTS) fromToTexts$: Observable<[string, string]>,
-    ) {
-        super(control, changeDetectorRef, documentRef, fromToTexts$);
-    }
+export class TuiSliderComponent {
+    @Input()
+    @HostBinding('attr.data-size')
+    @tuiDefaultProp()
+    size: TuiSizeS = 'm';
 
-    get nativeFocusableElement(): TuiNativeFocusableElement | null {
-        return this.dotRight ? this.dotRight.nativeElement : null;
-    }
+    @Input()
+    @tuiDefaultProp()
+    @HostBinding('class._ticks')
+    showTicks = false;
 
-    get focused(): boolean {
-        return isNativeFocused(this.nativeFocusableElement);
+    @HostBinding('attr.type')
+    readonly type = 'range';
```

Comment:
Maybe move it to `host` of `@Component` decorator since it's never going to change?